packages/libsql/src/lib/LibSqlClient.ts

```diff
@@ -131,13 +137,14 @@ export class LibSqlReader {
     return result.rows.map(Parse.row)
   }
 
-  async readSnapshot(streamName: string) {
+  async readSnapshot(streamName: string): Promise<[string, ITimelineEvent] | undefined> {
     const result = await this.client.execute({
       sql: "select * from snapshots where stream_name = ?",
       args: [streamName],
     })
     if (result.rows.length === 0) return
```

Comment:
I wonder is there a linting rule you can set to require `return undefined` to be explicitly written to make people think?
Or is it just that in TS, thinking aint optional, ever!

Reply:
`return` is equivalent to `return undefined`, what's the problem?

Reply:
Yes, was aware they are equivalent
Was just pondering that if it forced you to type `return undefined` that it would be more scannable
But it's not like sane scannability is in the top 10 concerns of JS, so why should it be for TS